app/js/app/directives/react_classes/MetaData.js

```diff
@@ -371,8 +371,8 @@ define(["react", "jquery"], function(React,$) {
 					</div>;
 				}
 
-				if (this.props.doc.type == "emailTemplate") {
-					var emailTemplateMeta = [
+				if (this.props.doc.type == "emailTemplate" || this.props.doc.type == "isaacPod") {
+					var published = [
```

Comment:
What is this change for?

Reply:
It only contains the published option, and we're now using it for pods as well as emails, so I made the name a bit more generic